.github/workflows/script/PrLabeled.ps1

```diff
@@ -0,0 +1,50 @@
+param(
+    [System.String]
+    $LabelName,
+    [System.String]
+    $PrUrl
+)
+Write-Host "Processing PR $PrUrl with label $LabelName"
+
+$CommentDict = @{
+    "Breaking Change Release" = "Azure PowerShell breaking change policy, please refer to [Breaking Change Definition](https://eng.ms/docs/cloud-ai-platform/azure-core/azure-management-and-platforms/control-plane-bburns/azure-cli-tools-azure-cli-powershell-and-terraform/azure-cli-tools/devguide/azps/breaking-change/breaking-changes-definition)"
```

Comment:
Let's discuss the format of the message. IMO
1. It should be clear which label is responsible for the message, for example, "(This comment is triggered by the xxx label)."
2. Let's use words like "Action required" or "Note" to highlight the severity.